src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinSpongeUser.java

```diff
@@ -93,11 +100,77 @@ public String getIdentifier() {
         return this.profile.getId().toString();
     }
 
+    private StatisticsFile getStatisticFile() {
+        MinecraftServer server = MinecraftServer.getServer();
+        File statsDir = new File(server.worldServerForDimension(0).getSaveHandler().getWorldDirectory(), "stats");
+        File statsFile = new File(statsDir, getUniqueId().toString() + ".json");
```

Comment:
Are you referring to a `statsFile#exists()` check? This is done internally already.